hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/client/validator/SparkPreCommitValidator.java

```diff
@@ -70,7 +70,7 @@ protected Set<String> getPartitionsModified(HoodieWriteMetadata<O> writeResult)
    * Throw HoodieValidationException if any unexpected data is written (Example: data files are not readable for some reason).
    */
   public void validate(String instantTime, HoodieWriteMetadata<O> writeResult, Dataset<Row> before, Dataset<Row> after) throws HoodieValidationException {
-    HoodieTimer timer = new HoodieTimer().startTimer();
+    HoodieTimer timer = HoodieTimer.start();
```

Comment:
is there any other usage of deprecated `new HoodieTimer().startTimer()` in the project in other places? we can also update it in the PR.

Reply:
@leesf Thanks for your help reviewing the code, I will follow up.

Reply:
+1